### PR TITLE
Add a current class to clicked documents in the previewer for improve…

### PR DESCRIPTION
**Pull request by syndacks**
…d UX

### CODE REVIEW

File: previewer/main_renderer.js [hunk 1]
```diff
@@ -71,6 +71,15 @@ $(document).ready(function(){
               )
         )
       )
+
+      // give a 'curent' class to clicked items the documentList
+      $('tr').on('click', function(){
+        $(this)
+        .addClass('current')
+        .siblings('.current')
+        .removeClass('current');
```
Comment (sgnn7) [inline previewer/main_renderer.js]: I think there is toggleClass that you should use here.

**Comment by sgnn7** [comment]
@syndacks One small comment

**Comment by syndacks** [comment]
@sgnn7 forgot about this one! I made the fix per your comment one month and one day ago.

File: previewer/main_renderer.js [hunk 2]
```diff
@@ -71,6 +71,15 @@ $(document).ready(function(){
               )
         )
       )
+
+      // give a 'curent' class to clicked items the documentList
+      $('tr').on('click', function(){
+        $(this)
+        .addClass('current')
+        .siblings('.current')
```
Comment (sgnn7) [inline previewer/main_renderer.js]: Split this out for easier reading - one line for adding the class and one to remove it from other siblings.

**Comment by syndacks** [comment]
@sgnn7 ready for review again

**Comment by syndacks** [comment]
@sgnn7 no worries, thanks for being thorough.

**Comment by sgnn7** [comment]
CR Pass

**Pull request merged**